src/util.ts

```diff
@@ -0,0 +1,39 @@
+import type { Writable } from 'node:stream'
```

Comment:
Could you rename this file to `utils.ts`?

Reply:
Ok.